tests/rptest/tests/consumer_group_test.py

```diff
@@ -151,6 +155,13 @@ def test_basic_group_join(self, static_members):
             c.wait()
             c.free()
 
+        viewer = OfflineLogViewer(self.redpanda)
+        for node in self.redpanda.nodes:
+            consumer_offsets_records = viewer.read_consumer_offsets(node=node)
```

Comment:
This test knows how many consumer groups it expects to see, right?  Would be good to validate that in the output, rather than only checking that the parser doesn't throw.